src/backend/utils/storage.js

```diff
@@ -59,12 +59,12 @@ module.exports = {
       .exec();
   },
 
-  getPosts: counter =>
+  getPosts: (from, to) =>
```

Comment:
Can you add a JSDOC style comment above this, documenting the expected type/values for `from` and `to`?  People won't know if they are dates, scalar numbers, etc.